build/yaml/steps/windows/solutionbuild.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Step Template: Build Windows Binaries

```

Comment:
>Build Windows Binaries [](start = 17, length = 22)

This looks very similar to binaries.yaml, just without nuget restore. Can we consolidate the two?